public/js/chat.js

```diff
@@ -1,48 +1,38 @@
-const hamburger = document.getElementsByClassName('huhamburger')[0];
-const aside = document.querySelectorAll('aside .hide_aside')[0];
-const side = document.getElementsByTagName('aside')[0];
-const alter_header = document.getElementsByClassName('alter_header')[0];
-const header = document.getElementsByTagName('header')[0];
-const messages = document.querySelectorAll('.main_section .last_messages ul')[0];
-const messenger_space = document.querySelector(' main .main_section .messenger');
-const last_messages = document.querySelector('main .main_section .last_messages');
-const back_btn = document.querySelector('.alter_header i');
-
-if (hamburger) {
-  hamburger.addEventListener('click', () => {
-    side.classList.toggle('hide');
-    aside.classList.toggle('side');
-    aside.classList.toggle('hide_aside');
-  });
-}
+
+const header = document.getElementById('header');
+const messages = document.getElementById('messages');
+const messenger_space = document.getElementById('messenger_space');
+const last_messages = document.getElementById('last_messages');
+const back_btn = document.getElementById('back_btn');
 
 messages.addEventListener('click', (event) => {
+
   if (window.innerWidth < 750 && event.target.tagName !== 'UL') {
-    last_messages.setAttribute('style', 'display:none');
-    messenger_space.setAttribute('style', 'display:block');
-    header.setAttribute('style', 'display:none');
-    alter_header.setAttribute('style', 'display:flex');
+    headerVisible('none', 'block', 'none', 'flex');
   }
+
 });
+back_btn.addEventListener('click', () => {
 
-back_btn.addEventListener('click', (event) => {
   if (window.innerWidth < 750) {
-    last_messages.setAttribute('style', 'display:block');
-    messenger_space.setAttribute('style', 'display:none');
-    header.setAttribute('style', 'display:flex');
-    alter_header.setAttribute('style', 'display:none');
+    headerVisible('block', 'none', 'flex', 'none');
   }
-});
 
+});
 window.addEventListener('resize', () => {
+
   if (window.innerWidth > 750) {
-    last_messages.setAttribute('style', 'display:block');
-    messenger_space.setAttribute('style', 'display:block');
-    header.setAttribute('style', 'display:flex');
-    alter_header.setAttribute('style', 'display:none');
+    headerVisible('block', 'block', 'flex', 'none');
   } else {
     last_messages.setAttribute('style', 'display:block');
```

Comment:
line 27-29 
you could also use the headerVisible function.
you just need to check if the fourth parameter presented or not 

Also move the function expression to the head of the file or convert it to a function declaration   
check the [difference between function expression and declaration](https://stackoverflow.com/questions/1013385/what-is-the-difference-between-a-function-expression-vs-declaration-in-javascrip) 

Note--> it is better to name the function as a verb if it changes the state of the app something like `changeHeaderVisibility`
```js
const headerVisible = (last_messages, messenger_space, header, alter_header) => {
  last_messages.setAttribute('style', `display:${last_messages};`);
  messenger_space.setAttribute('style', `display:${messenger_space};`);
  header.setAttribute('style', `display:${header};`);
  if(alter_header)
       alter_header.setAttribute('style', `display:${alter_header};`);
};
```